kolibri/core/assets/src/views/PaginatedListContainerWithBackend.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+
+  <div>
+    <KGrid>
+      <KGridItem :layout12="{ span: 7 }">
+        <slot name="otherFilter"></slot>
+      </KGridItem>
+      <KGridItem
+        :layout12="{ span: 5, alignment: 'right' }"
+        class="text-filter"
+      >
+        <FilterTextbox v-model="filterInput" :placeholder="filterPlaceholder" />
+      </KGridItem>
+    </KGrid>
+
+    <div>
+      <slot
+        :items="userList"
+        :filterInput="filterInput"
+      >
+      </slot>
+    </div>
+
+    <nav class="pagination-nav">
+      <span dir="auto" class="pagination-label">
+        {{ $tr('pagination', { visibleStartRange, visibleEndRange, numFilteredItems }) }}
+      </span>
+      <KButtonGroup>
+        <KIconButton
+          :ariaLabel="$tr('previousResults')"
+          :disabled="previousButtonDisabled"
+          size="small"
+          icon="back"
+          @click="changePage(-1)"
+        />
+        <KIconButton
+          :ariaLabel="$tr('nextResults')"
+          :disabled="nextButtonDisabled"
+          size="small"
+          icon="forward"
+          @click="changePage(+1)"
+        />
+      </KButtonGroup>
+    </nav>
+  </div>
+
+</template>
+
+
+<script>
+
+  import clamp from 'lodash/clamp';
+  import FilterTextbox from 'kolibri.coreVue.components.FilterTextbox';
+  import { FacilityUserResource } from 'kolibri.resources';
+  import store from 'kolibri.coreVue.vuex.store';
+  import { _userState } from '../../../../plugins/facility/assets/src/modules/mappers';
```

Comment:
This `_userState` mapper is making this component specific to display in the facility plugin context. As such, it should be removed from the core API and moved into the facility plugin specifically.

This makes even more sense when you consider that anything loaded into the coreAPI is loaded on every single page in Kolibri, but this is only used in the facility plugin.